src/reference-guide/controller-api/v2.3.md

```diff
@@ -576,6 +576,67 @@ DEIS_API_VERSION: 2.3
 DEIS_PLATFORM_VERSION: 2.3.0
 ```
 
+### Enable or disable TLS
+
+Example Request:
+
+```
+POST /v2/apps/example-go/tls/ HTTP/1.1
+Host: deis.example.com
+Content-Type: application/json
+Authorization: token abc123
+
+{
+  "https_enabled": true
+}
+```
+
+Example Response:
+
+```
+HTTP/1.1 201 CREATED
+DEIS_API_VERSION: 2.3
+DEIS_PLATFORM_VERSION: 2.3.0
+Content-Type: application/json
+
+{
+    "created": "2014-01-01T00:00:00UTC",
+    "app": "example-go",
+    "owner": "test",
+    "https_enforced": true,
+    "updated": "2014-01-01T00:00:00UTC",
+    "uuid": "de1bf5b5-4a72-4f94-a10c-d2a3741cdf75"
+}
+```
+
+### Get TLS status
+
+Example Request:
+
+```
+GET /v2/apps/example-go/tls/ HTTP/1.1
+Host: deis.example.com
+Authorization: token abc123
+```
+
+Example Response:
+
+```
+HTTP/1.1 200 OK
+DEIS_API_VERSION: 2.3
+DEIS_PLATFORM_VERSION: 2.3.0
+Content-Type: application/json
+
+{
+    "created": "2014-01-01T00:00:00UTC",
+    "app": "example-go",
+    "owner": "test",
+    "https_enforced": false,
```

Comment:
The server code effectively coerces `null` to `false`. Once you've explicitly disabled `deis tls` I guess it will say `false` there. Should we change the docs to say `null`?